src/index.ts

```diff
@@ -61,6 +60,8 @@ import { getDataFromExternalSources } from './lib/getDataFromExternalSources';
 import { DynamicKeyParts } from './types/dynamicKeys';
 import { getData } from './lib/getData';
 
+const HttpProvider = require('web3-providers-http');
```

Comment:
Can we use import for this?

Reply:
I tried but was having weird issues
<img width="1037" alt="Screenshot 2022-09-29 at 22 52 12" src="https://user-images.githubusercontent.com/64636974/193220916-b8cf40f9-8db7-4698-9ea6-00bf82ebbfc6.png">
<img width="599" alt="Screenshot 2022-09-29 at 23 00 33" src="https://user-images.githubusercontent.com/64636974/193220919-939d430b-a6ae-4c13-93f5-7775838893f0.png">
<img width="535" alt="Screenshot 2022-09-29 at 23 02 56" src="https://user-images.githubusercontent.com/64636974/193220923-e6f270f9-6ab8-4f52-b7a8-60f56f10ecb9.png">